dbms/src/Common/SensitiveDataMasker.cpp

```diff
@@ -61,15 +64,47 @@ class SensitiveDataMasker::MaskingRule
     uint64_t apply(std::string & data) const
     {
         auto m = RE2::GlobalReplace(&data, regexp, replacement);
+#ifndef NDEBUG
         matches_count += m;
+#endif
         return m;
     }
 
     const std::string & getName() const { return name; }
     const std::string & getReplacementString() const { return replacement_string; }
+#ifndef NDEBUG
     uint64_t getMatchesCount() const { return matches_count; }
+#endif
+
 };
 
+bool SensitiveDataMasker::sensitive_data_masker_initialized = false;
```

Comment:
Can we just check for that the sensitive_data_masker = nullptr?

Reply:
It can be null after initialization too. For example - there is a corresponding section in config, but no rules there

Reply:
So is it a debugging aid to check that the masker is initialized once? I don't see much value in it, let's remove.